src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/androids/WoodcutterAndroid.java

```diff
@@ -69,13 +69,14 @@ private void breakLog(Block log, Block android, BlockMenu menu, BlockFace face)
 
         if (menu.fits(drop, getOutputSlots())) {
             menu.pushItem(drop, getOutputSlots());
-            log.getWorld().playEffect(log.getLocation(), Effect.STEP_SOUND, log.getType());
+        }
 
-            if (log.getY() == android.getRelative(face).getY()) {
-                replant(log);
-            } else {
-                log.setType(Material.AIR);
-            }
+        log.getWorld().playEffect(log.getLocation(), Effect.STEP_SOUND, log.getType());
+
+        if (log.getY() == android.getRelative(face).getY()) {
+            replant(log);
+        } else {
+            log.setType(Material.AIR);
         }
```

Comment:
This will void a bit of logs sadly but better than the alternative I guess.

Some comments here would be nice.